.github/workflows/changelog.yml

```diff
@@ -8,8 +8,11 @@ on:
 
 jobs:
   community_check:
+    if: github.event_name == 'pull_request_target'
```

Comment:
This one has a similar idea as is mentioned in my comment on `autoremove_labels.yml`, but has a bit of an interesting caveat that I'd like to look into more.

Previously, this conditional was in the `changes` job. This meant that if the event type was `push`, the `changes` job would be skipped, which would caused the `comment` job to be skipped as well, since `changes` is in its `needs`. This means that `push` events never trigger any of the jobs in this workflow.

Reply:
In looking at this further, the `comment` job also uses `github.event.pull_request.number`, which isn't included in `push` events, so it seems safe to remove the `on.push` triggers entirely. Will wait for review before doing so, in case I'm overlooking anything.